staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/config/apis/webhookadmission/install/install.go

```diff
@@ -31,11 +32,12 @@ func Install(registry *registered.APIRegistrationManager, scheme *runtime.Scheme
 	if err := announced.NewGroupMetaFactory(
 		&announced.GroupMetaFactoryArgs{
 			GroupName:                  webhookadmission.GroupName,
-			VersionPreferenceOrder:     []string{v1alpha1.SchemeGroupVersion.Version},
+			VersionPreferenceOrder:     []string{v1.SchemeGroupVersion.Version, v1alpha1.SchemeGroupVersion.Version},
```

Comment:
I'm not sure about this change because v1 has one type and v1alpha1 has others, in other words these aren't different version of the same API but different.

Reply:
The admissionregistration seems to do something similar where different versions of the API hold different object.

https://github.com/kubernetes/api/blob/kubernetes-1.10.2/admissionregistration/v1alpha1/register.go#L45-L48
https://github.com/kubernetes/api/blob/kubernetes-1.10.2/admissionregistration/v1beta1/register.go#L45-L50

Any apimachinery folk care to comment here? It looks fine to me.